zio-http/src/main/scala/zhttp/http/Cookie.scala

```diff
@@ -125,6 +129,29 @@ final case class Cookie(
     cookie.flatten.mkString("; ")
   }
 
+  def sign(secret: String): Option[Cookie] = {
```

Comment:
```suggestion
  def sign(secret: String): Cookie = {
```

`sign` should add the secret in the cookie.